### DISCUSSION TITLE
How to disable wingetui updates?

### DISCUSSION THREAD

**Discussion by Rapunzel77** [Devolutions/UniGetUI#1947] [Q&A] [answered]
I have added WingetUI to the "ignore updates" list. I have some trouble with the version 3 that just came out (cloud sniper is killing it).
I downgraded (to 2.2.0) and added it to the "ignore list" in the UI but it will prompt for installation every time I open WingetUI. I simply click "cancel" every time. But that seems to not be a good solution.
Any ideas?

**Comment by marticliment** [comment, marked as answer]
You can disable automatic updates from WingetUI Settings

**Reply by Rapunzel77** [reply]
ahh yes - thanks. I guess that resolves this. I just thought that it would follow the "ignore update" in the global list :)
